rackcorp/api/order.go

```diff
@@ -35,6 +45,7 @@ type ProductDetails struct {
 	MemoryGB         int              `json:"memoryGB"`
 	TrafficGB        int              `json:"trafficGB,omitempty"`
 	FirewallPolicies []FirewallPolicy `json:"firewallPolicies,omitempty"`
+	Nics             []Nic            `json:"nics,omitempty"`
```

Comment:
I feel casing should be `NICs`

Reply:
Whilst I personally disagree with the emerging go conventions - I support little minds' hobgoblins.
https://github.com/golang/go/wiki/CodeReviewComments#initialisms